### ISSUE TITLE
[Bug]: Popover content is appearing in a different position from target (React)

### ISSUE THREAD

**Issue by mounikanukala29** [GovAlta/ui-components#1988]
### Describe the bug

When the window size is changed, the popover content is appearing in a different position from the target.

### To Reproduce

Resize the windows and check the position of the popover content keep moving away from the Goa PopOver button[
![bug](https://github.com/GovAlta/ui-components/assets/175065568/d57fd829-3c76-4698-b87b-7f07a5faf6fa)
](url)

### What browsers are you seeing the problem on?

Chrome, Edge

**Comment by ArakTaiRoth** [comment]
Spoke to Mounika about this, her package versions are out of date. Asking her to update and report back if this is still an issue.

**Comment by ArakTaiRoth** [comment]
I don't believe this is an issue any longer, I can't seem to create any replication for this now. @willcodeforcoffee Can you try please?
